.ebextensions/deploy.config

```diff
@@ -10,3 +10,22 @@ commands:
         command: "/opt/python/run/venv/bin/pip  install --upgrade pip"
     02-install-cryptography:
         command: "source /opt/python/run/venv/bin/activate; pip install -e git+https://github.com/ONSdigital/cryptography.git@4a90c254278231d7defeac304a3cfd752e96e786#egg=cryptography"
+
+files:
+    "/etc/logrotate.elasticbeanstalk.hourly/logrotate.elasticbeanstalk.httpd.conf":
+        mode: "000644"
+        owner: root
+        group: root
+        content: |
+           /var/log/httpd/* {
+                size 10M
+                missingok
+                notifempty
+                rotate 5
+                sharedscripts
+                compress
+                dateext
+                dateformat -%s
+                copytruncate
```

Comment:
No, I think we need to take another look at this. I have separated out the log format change from this in #1029